zero_bin/leader/Cargo.toml

```diff
@@ -30,10 +30,12 @@ toml = { workspace = true }
 ops = { workspace = true }
 prover = { workspace = true }
 rpc = { workspace = true }
+evm_arithmetization = { workspace = true }
```

Comment:
I think is, because we link the cdk_erigon feature in the leader to the one in evm_arithmetization... (If I remove it, it says the evm_arithmetization feature is not found). Maybe there's a way around this though...